src/projectM-emscripten/README

```diff
@@ -1,27 +1,39 @@
 Running on the web.
 
-# Easy way:
+# TLDR:
+* `./autogen.sh`
+* `emconfigure ./configure --enable-emscripten --enable-gles --enable-sdl`
+* `emmake make -j4 -lvendor`
+* `cd src/projectM-emscripten`
 * `make run`
 
 
-
-
-# Hard way:
+# Detailed:
 
 ## Prepare
 * Activate the emsdk (https://github.com/juj/emsdk#installation-instructions)
 * make sure you are in the root directory of this project
 * on fresh repositories: `./autogen.sh`
+* download latest glm from https://github.com/g-truc/glm/releases/latest
+* make sure to move it to `./vendor/glm`
 
 ## Compile
 * `emconfigure ./configure --enable-emscripten --enable-gles --enable-sdl`
-* `emmake make -j4`
+* `emmake make -j4 -lvendor`
 
 ## Create wasm & html files
 * `cd src/projectM-emscripten`
 * `emcc -s USE_SDL=2 -s ALLOW_MEMORY_GROWTH=1 projectW.bc -o projectW.html`
 
-Does not work? Try:
+## Troubleshooting
 
-* `mkdir vendor && cp /usr/local/Cellar/glm/xx/include/glm ./vendor/`
-* `emmake make -j4 -lvendor`
+### General
+
+Want to restart the process after pulling or changing config?
+* `rm src/projectM-emscripten/projectW*`
```

Comment:
should be a clean target

Reply:
Is there any way to ensure this like a cleanup script?